pages/1.10/metrics/prometheus/index.md

```diff
@@ -3,51 +3,80 @@ layout: layout.pug
 title: Sending DC/OS Metrics to Prometheus
 menuWeight: 4.5
 excerpt:
-preview: true
 enterprise: false
 ---
 
-The Prometheus metrics plugin supports sending metrics from the DC/OS metrics service to a [Prometheus](https://prometheus.io/) server. You must install a plugin on each node in your cluster. This plugin works with DC/OS 1.9.4 and higher.
+The Prometheus metrics plugin supports sending metrics from the DC/OS metrics service to a [Prometheus](https://prometheus.io/) server. You must install a plugin on each node in your cluster. 
 
 **Prerequisite:**
 
-- DC/OS is [installed](/1.10/installing/oss/)
+- You must have the [DC/OS CLI installed](/1.10/cli/install/) and be logged in as a superuser via the `dcos auth login` command.
 
 # Install the DC/OS Prometheus metrics plugin
 
-For each node in your cluster, transfer your plugin binary and then add a systemd unit to manage the service. This unit differs slightly between master and agent nodes.
+For each node in your cluster, download the plugin binary and then add a systemd unit to manage the service. This unit differs slightly between master and agent nodes.
 
 1. On every node in your cluster:
 
-   1. Download the _latest_ Prometheus plugin binary from the [releases](https://github.com/dcos/dcos-metrics/releases) page.
-   1. Rename the plugin to `dcos-metrics-prometheus-plugin` and move to `/opt/mesosphere/bin`.
-   1. Assign permissions to the plugin: `chmod 0755 /opt/mesosphere/bin/dcos-metrics-prometheus-plugin`.
-   1. Download the [environment file](https://raw.githubusercontent.com/dcos/dcos-metrics/master/plugins/prometheus/systemd/dcos-metrics-prometheus.env) to `/opt/mesosphere/etc`. This file sets the port to 8088. (8088 was chosen because 8080 is the standard, but was already reserved for Marathon). To serve on a different port, edit the environment file and set `PROMETHEUS_PORT`.
-   1. Set the environment variable `DCOS_METRICS_CONFIG_PATH` to `/opt/mesosphere/etc`.
+   1. Download the latest Prometheus plugin binary from downloads.mesosphere.io: [prometheus-plugin](https://downloads.mesosphere.io/dcos-metrics/plugins/prometheus)
+   1. Rename the plugin to `dcos-metrics-prometheus` and move to `/opt/mesosphere/bin`.
+   1. Assign permissions to the plugin: `chmod 0755 /opt/mesosphere/bin/dcos-metrics-prometheus`.
 
 1.  On every master node:
-    1. Download the master [systemd service file](https://raw.githubusercontent.com/dcos/dcos-metrics/master/plugins/prometheus/systemd/dcos-metrics-prometheus-master.service) to `/etc/systemd/system`.
+    1. Download the plugin systemd service file from downloads.mesosphere.io: [prometheus-plugin.service](https://downloads.mesosphere.io/dcos-metrics/plugins/prometheus.service)
```

Comment:
The unit file at that location contains this line:
`ExecStart=/opt/mesosphere/bin/dcos-metrics- --dcos-role master --prometheus-port 61091`

Notice that 'prometheus' is missing from the executable name...  You mention the correct name below but it seems like it would be better if the file you downloaded had the correct name to start with.

Reply:
@philipnrmn 
You need to add a "period" after  the below line
 1. Download the latest Prometheus plugin binary from downloads.mesosphere.io: [prometheus-plugin](https://downloads.mesosphere.io/dcos-metrics/plugins/prometheus)

Reply:
@philipnrmn in line number 34 and 41, you can rewrite this 
"On every master node:" as "On every master node do the following steps:"
"On every agent node:" as "On every agent node do the following steps:"

Line 59: Rewrite 
There are many ways to run a Prometheus server. This is the simplest way to get started with self-hosted metrics on DC/OS. 
as
" There are several ways to run a Prometheus server. The simplest way to get started with self-hosted metrics on DC/OS is listed below:"